iot-care-pet-example/go/README.md

```diff
@@ -0,0 +1,107 @@
+Care Pet example
+===
+
+This is an example project that demonstrates basic usage
+patterns of ScyllaDB Go driver infrastructure.
+
+The application allows tracking of pets health indicators
+and consist of 3 parts:
+
+- migrate (`/cmd/migrate`) - bootstrap database schema
+- collar sensor (`/cmd/sensor`) - accumulates pet health data and publish it
+- web app (`/cmd/server`) - REST API for tracking pets state
+
+Quick Start
+---
+
+Compose will spin 3 ScyllaDB nodes pinned to the first core.
+
+    $ docker-compose up
+    $ docker exec -it carepet-scylla1 cqlsh
+
+To initialize database execute:
+
+    $ go build ./cmd/migrate
+    $ ./migrate --hosts carepet-scylla1
```

Comment:
I got the following error
```
./migrate --hosts carepet-scylla1
2020/08/06 10:04:56 Bootstrap database...
2020/08/06 10:04:57 gocql: dns error: lookup carepet-scylla1 on 192.168.0.1:53: no such host
2020/08/06 10:04:57 session:  gocql: unable to create session: failed to resolve any of the provided hostnames
```

Altho this did work
```
./migrate --hosts carepet-scylla1
2020/08/06 10:04:56 Bootstrap database...
2020/08/06 10:04:57 gocql: dns error: lookup carepet-scylla1 on 192.168.0.1:53: no such host
2020/08/06 10:04:57 session:  gocql: unable to create session: failed to resolve any of the provided hostnames
```

Reply:
Rewrote into:

```
To initialize database execute:

    $ go build ./cmd/migrate
    $ NODE1=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' carepet-scylla1)
    $ ./migrate --hosts $NODE1
    
    expected output:
    
    2020/08/06 16:43:01 Bootstrap database...
    2020/08/06 16:43:13 Keyspace metadata = {Name:carepet DurableWrites:true StrategyClass:org.apache.cassandra.locator.NetworkTopologyStrategy StrategyOptions:map[datacenter1:3] Tables:map[gocqlx_migrate:0xc00016ca80 measurement:0xc00016cbb0 owner:0xc00016cce0 pet:0xc00016ce10 sensor:0xc00016cf40 sensor_avg:0xc00016d070] Functions:map[] Aggregates:map[] Types:map[] Indexes:map[] Views:map[]}
```